dashboard/test/controllers/level_sources_controller_test.rb

```diff
@@ -167,22 +167,6 @@ def expect_s3_upload
     assert_equal "max-age=36000, public", response.headers["Cache-Control"]
   end
 
-  test 'include level source ID for send to phone dialog' do
-    # Prevents regressions in #79201066
```

Comment:
AFAICT, appOptions.level_source_id isn't used by the share dialog anymore, and the dialog works for me locally with artist and applab. Is there some other project type that could still be using it?